src/erlcloud_ddb_streams.erl

```diff
@@ -217,6 +217,9 @@ undynamize_value_untyped({<<"BS">>, Values}, _) ->
     [base64:decode(Value) || Value <- Values];
 undynamize_value_untyped({<<"L">>, List}, Opts) ->
     [undynamize_value_untyped(Value, Opts) || [Value] <- List];
+undynamize_value_untyped({<<"M">>, [{}]}, _Opts) ->
+    %% jsx returns [{}] for empty objects
+    [];
```

Comment:
@dfedyaschin so no use of `#{}` ?

Reply:
Maps are proplists everywhere else in erlcloud, I wish they were actual maps...

Reply:
Someday!